tests/testthat/test-names.R

```diff
@@ -1,4 +1,5 @@
 test_that("names", {
+  skip("not allowing invalid names")
```

Comment:
work in progress, forgotten or do the tests no longer work?

Reply:
Oh, I mentioned it in in #165 but it's a little vague.  The intention is to not allow any illegal characters in the name and instruct the user to explicitly remove them. `private$validate_new_sheet()` will throw an error when illegal character names are passed.  Prior the name checks would quietly change the names, but this will make it more intentional.

Reply:
I see, makes sense. Thanks for the explanation.